### PR TITLE
🍊 Tabs on sidebar

### LINKED ISSUE #65: Create context tabs
**Summary**
When a user clicks a tab, we should be able to:
* change sidebar content
* change the context of the map (create a state variable)

**Consult Siraj for latest designs**
image

### PR DESCRIPTION
**Pull request by lucianatl**
### Changes

- added arrest, massage parlor laws, vacatur laws and criminal law tabs on sidebar
- tried to add orange bar at the bottom to match the Figma and make it look like tabs, however yarn stopped working before I was able to finish that part so I currently I have no idea how it looks 
- no screenshots even though there was a a UI change bc of above problem 

---

### Screenshots (if UI change)

---

### Checklist

- [x] Added a11y fields (if applicable)

### CODE REVIEW

File: client/src/components/SidebarContainer.jsx [hunk 1]
```diff
@@ -183,7 +186,56 @@ const SidebarContainer = (props: Props) => {
...
+          style={{ 'background-color': tab === 1 ? 'darkorange' : 'grey' }}
+          onClick={() => (setTab(1))}>
+          Massage Parlor Laws</button>
+        <button class="tablinks bg-orange-500 text-center text-white font-sans w-1/3 py-2 px-4 text-xs rounded"
+          aria-label="Vacatur Laws"
+          style={{ 'background-color': tab === 2 ? 'darkorange' : 'grey' }}
+          onClick={() => (setTab(2))}>
+          Vacatur Laws</button>
+        <button class="tablinks bg-orange-500 text-center text-white font-sans w-1/3 py-2 px-4 text-xs rounded"
+          aria-label="Criminal Law"
```
Comment (rxun) [inline client/src/components/SidebarContainer.jsx]: nit: change "Criminal Law" to "Criminal Laws"

**Review by rxun — APPROVED** [review]
good work! the ui is functional, but just clean up some of the design (and the small nit i ahd)
my concern comes with whether or not tab changes will be accessible since visibility / toggling states doesn't necessarily alert the user in react, i think?? don't worry about it now, i'll place it into the running list of a11y things we'd like to look over later

File: client/src/components/SidebarContainer.jsx [hunk 2]
```diff
@@ -37,6 +37,9 @@ const SidebarContainer = (props: Props) => {
 
   const [years, setYears] = useState([]);
   const [arrestData, setArrestData] = useState(null);
+  const [tab, setTab] = useState(0);
```
Comment (kenetec) [inline client/src/components/SidebarContainer.jsx]: Can we move tab to `MapPage.jsx` and make constants for all values of the tab var (0...3), that way we can update the map when the user switches tabs.

File: client/src/components/SidebarContainer.jsx [hunk 3]
```diff
@@ -183,7 +186,56 @@ const SidebarContainer = (props: Props) => {
           </h2>
         </div>
       </div>
-      <SidebarChart arrests={null} laws={null} />
+
+
+
+      <div class="tab flex flex-row m-1 pt-3 pb-1" style={{ 'border-bottom': '1.5em solid orange' }}>
+        <button class="tablinks bg-orange-500 text-center text-white font-sans  w-1/3 py-2 px-4 text-xs rounded"
+          style={{ 'background-color': tab === 0 ? 'darkorange' : 'grey' }}
```
Comment (kenetec) [inline client/src/components/SidebarContainer.jsx]: Replace all comparisons with `tab` with constants

**Review by kenetec — CHANGES_REQUESTED** [review]
some small changes for the code

**Review by kenetec — APPROVED** [review]
littyyy 🤩💯

**Pull request merged**